<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinUserManagerWidgetUI</class>
 <widget class="QWidget" name="WinUserManagerWidgetUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxWinUsersList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Users</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTableView" name="tableViewUsers">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>2</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="toolTip">
         <string>Users</string>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionCreateNewAccount">
   <property name="text">
    <string>Create New Account</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionDeleteAccount">
   <property name="text">
    <string>Delete Account</string>
   </property>
   <property name="toolTip">
    <string>Delete Account</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionLogoff">
   <property name="text">
    <string>Logoff</string>
   </property>
  </action>
  <action name="actionLockDesktop">
   <property name="text">
    <string>Lock Desktop</string>
   </property>
  </action>
  <action name="actionSendMessage">
   <property name="text">
    <string>Send Message</string>
   </property>
  </action>
  <action name="actionMonitorDesktop">
   <property name="text">
    <string>Monitor Desktop</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tableViewUsers</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
